core/src/test/java/com/opensymphony/xwork2/validator/DefaultActionValidatorManagerTest.java

```diff
@@ -87,7 +87,7 @@ protected void tearDown() throws Exception {
 
 
     public void testBuildValidatorKey() {
-        String validatorKey = DefaultActionValidatorManager.buildValidatorKey(SimpleAction.class, alias);
+        String validatorKey = actionValidatorManager.buildValidatorKey(SimpleAction.class, alias);
```

Comment:
Changed this to an instance method for consistency between the 2 implementations